pkg/operation/botanist/component/logging/kube_rbac_proxy.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	managedResourceName = "shoot-node-logging"
+	kubeRBACProxyManagedResourceName = "shoot-node-logging"
 
```

Comment:
kube rbac proxy could be moved into an own sub package too, as we introduce sub packages for logging now (see first suggestion).